app/models/spotify_track_getter.rb

```diff
@@ -0,0 +1,51 @@
+class SpotifyTrackGetter
+  RANDOM_TRACKS = {
+    breathe_the_fire: "7qLDBd5iCh9xilyfdkn9v8",
+    chinatown_style: "0KisC636mzjXthrEjmebyZ",
+    dark_entries: "3tuaOXGEQ8FigsBxCVs6ym",
+    dipped_into_poison: "3rTt87UHznxregyc3chpHD",
+    do_it_myself: "5mB1bWU8VueYVxuvcLPn2B",
+    dont_fear_the_reaper: "5QTxFnGygVM4jFQiBovmRo",
+    egyptian_reggae: "3FtqoENKDN0nq5bWNxHPYR",
+    electric_current: "2JCmUdoi8YBXamZRvZD5ms",
+    psychic_trauma: "6mUWyM02M91VoRGKFeRhjt",
+  }.freeze
+
+  def self.send_spotify_request
+    token_url = "https://accounts.spotify.com/api/token"
+    grant_type = { grant_type: "client_credentials" }
+    authorization_basic = { "Authorization": "Basic #{client_token}" }
+
+    @spotify_token = JSON.parse(
+      RestClient.post(token_url, grant_type, authorization_basic),
+    )
+  end
+
+  def self.get_spotify_tracks
+    track_id = random_track
+    track_url = "https://api.spotify.com/v1/tracks/#{track_id}"
+    access_token = @spotify_token.values[0]
+    authorization_bearer = { "Authorization": "Bearer #{access_token}" }
+
+    @track = JSON.parse(RestClient.get(track_url, authorization_bearer))
+  end
+
+  def self.client_token
+    Base64.strict_encode64(
+      [SPOTIFY_CLIENT_ID, SPOTIFY_CLIENT_SECRET].join(":"),
+    ).delete("\n")
+  end
+
+  def self.random_track
+    RANDOM_TRACKS.values.sample
+  end
+
+  def self.track_data
+    {
+      image: @track.values[0].values[6][1].values[1],
```

Comment:
These numbers make me nervous. How do you feel about

- validating `@track` against a JSON schema before using it, and
- naming the numbers (e.g. `ALBUM_INFO = 0`)?